addons/context_actions/CfgContext.hpp

```diff
@@ -199,6 +199,29 @@ class EGVAR(context_menu,actions) {
         statement = QUOTE(_hoveredEntity call EFUNC(garage,openGarage));
         priority = -82;
     };
+    class HealUnits {
+        displayName = ECSTRING(modules,ModuleHeal);
+        icon = QPATHTOF(ui\medical_cross_ca.paa);
+        condition = QUOTE(_selectedObjects findIf {crew _x findIf {alive _x} != -1} != -1);
```

Comment:
The action(s) shows when a module is in the selection. I think this needs a `_x isKindOf 'CAManBase'` check.